### PR TITLE
Create Event Components

### PR DESCRIPTION
**Pull request by Lanceart**
Hi, I finished the event bars part.

### CODE REVIEW

**Comment by pondohoo** [comment]
can you link the related issue under "development," add the labels associated with the issue, and post an image of your work running on a page?

File: src/app/page.js [hunk 1]
```diff
@@ -1,8 +1,21 @@
 import Landing from "@/components/home/Landing";
-
+import Event from "@/components/events/Event";
 const home = () => {
   return (
     <div>
+      <div className="relative py-3 sm:max-w-xl sm:mx-auto">
```
Comment (pondohoo) [inline src/app/page.js:6]: it's ok to add the component to the page for seeing what it looks like but make sure you remove it before you push

File: src/components/events/Event.jsx [hunk 2]
```diff
@@ -1,5 +1,65 @@
...
+
+  return (
+    <div className="flex rounded-lg overflow-hidden shadow-md max-w-lg">
+      <div className={`${info.color} p-4 text-white text-center w-1/3`}>
+        <div className="text-2xl font-bold">{info.month}</div>
+        <div className="text-4xl">{info.day}</div>
+        <div className="text-xl">{info.time}</div>
+      </div>
+      <div className="flex-1 pl-4">
+        <input
```
Comment (pondohoo) [inline src/components/events/Event.jsx:38]: this shouldn't be an input and shouldn't have state, it's just a description box

File: src/components/events/Event.jsx [hunk 3]
```diff
@@ -1,5 +1,65 @@
...
+"use client";
+import React, { useState } from "react";
+const Event = ({
+  month = "FEB",
+  day = "29",
+  time = "7:00PM",
+  title = "Event Title",
+  link = "http://google.com",
+  description = "Lorem ipsum dolor hello this is a test for the lol",
+  color = "bg-orange-400",
```
Comment (pondohoo) [inline src/components/events/Event.jsx:10]: make sure you're using the colors specified under tailwind.config.js (bg-hlg-orange instead of bg-orange-400)

**Comment by Lanceart** [comment]
Oh, I just push a new version and please check that. Thank you!

File: src/components/events/Event.jsx [hunk 4]
```diff
@@ -1,5 +1,35 @@
...
+        <div className="text-4xl ">{day}</div>
+        <div className="text-xl">{time}</div>
+      </div>
+      <div className="flex-1 pl-4">
+        <p className="w-full text-xl font-bold mb-1 focus:outline-none resize-none font-bold">
+          {title}
+        </p>
+        <a
+          href={`${link}`}
+          className="w-full text-blue-500 mb-2 focus:outline-none resize-none"
```
Comment (pondohoo) [inline src/components/events/Event.jsx:25]: make text hlg-black like in the figma design

File: src/components/events/Event.jsx [hunk 5]
```diff
@@ -1,5 +1,35 @@
...
+        >
+          zoom: link
+        </a>
```
Comment (pondohoo) [inline src/components/events/Event.jsx:28]: next to the word "zoom:link" include the externallink react icon. check out the figma design

https://react-icons.github.io/react-icons/search?q=FiExternalLink

File: src/components/events/Event.jsx [hunk 6]
```diff
@@ -1,5 +1,35 @@
...
+  day = "29",
+  time = "7:00PM",
+  title = "Event Title",
+  link = "http://google.com",
+  description = "Lorem ipsum dolor hello this is a test for the lol",
+  color = "bg-hlg-orange",
+}) => {
+  return (
+    <div className="flex rounded-lg overflow-hidden shadow-md max-w-lg">
+      <div className={`${color} p-4 text-white text-center w-1/3`}>
```
Comment (pondohoo) [inline src/components/events/Event.jsx:14]: make the date text more bold to look like the figma design

**Comment by Lanceart** [comment]
Still remain the font-bold bug.

File: src/app/page.js [hunk 7]
```diff
@@ -1,8 +1,9 @@
 import Landing from "@/components/home/Landing";
-
+import Event from "@/components/events/Event";
 const home = () => {
   return (
     <div>
+      <Event />
```
Comment (pondohoo) [inline src/app/page.js:6]: it's ok to have this on the home page to test your issue but make sure you take it out before you commit

**Review by pondohoo — CHANGES_REQUESTED** [review]

File: src/app/page.js (hunk 7 above)
Comment (Lanceart) [reply src/app/page.js:6]: Oh, I am so sorry for this.  Please check it again. Next time I gonna check it carefully. Thank you!

**Review by pondohoo — APPROVED** [review]

**Pull request merged**